libr/core/cmd_anal.c

```diff
@@ -5961,8 +6010,7 @@ static void cmd_anal_graph(RCore *core, const char *input) {
 		case 't': { // "agft" - tiny graph
 			int e = r_config_get_i (core->config, "graph.edges");
 			r_config_set_i (core->config, "graph.edges", 0);
-			ut64 off_fcn = (*(input + 2)) ? r_num_math (core->num, input + 2) : core->offset;
-			RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, off_fcn, 0);
+			RAnalFunction *fcn = r_anal_get_fcn_in (core->anal, core->offset, 0);
```

Comment:
why drop the use of specifying the offset as argument?

Reply:
I think it makes more sense to have a consistent api that uses `@ addr` to seek to address, instead of having arguments for the seek.